components/dropdown/script.js

```diff
@@ -112,16 +115,17 @@ class GamefaceDropdown extends CustomElementValidator {
      * @param {string} value
     */
     set value(value) {
-        if (!value) {
-            this.setSelectedAndScroll(null);
+        const option = Array.from(this.allOptions).find(option => option.value === value);
+
+        if (!option && value === '') {
```

Comment:
Can we trim the value e.g. `value.trim()` since if someone by any chance enters ' ' with a space ('' !== ' ') it will show the console.warn?